modules/packages/ZMQ.chpl

```diff
@@ -167,12 +167,12 @@ Serialization
 +++++++++++++
 
 In Chapel, sending or receiving messages is supported for a variety of types.
-Primitive numeric types and strings are supported as the foundation.
-In addition, user-defined ``record`` types may be serialized automatically
-as `multipart messages <http://zguide.zeromq.org/page:all#Multipart-Messages>`_
-by internal use of the :chpl:mod:`Reflection` module.
-Currently, the ZMQ module can serialize records of primitive numeric types,
-strings, and other serializable records.
+Primitive numeric types, along with ``string`` and ``bytes`` are supported as the
```

Comment:
Need a comma after `''bytes''`. Or, you can go simple and say "... types, ``strings``, and ``bytes`` are...", and keep the rest of the paragraph unchanged - which IMO looks better than with your change.

Reply:
Yeah, I didn't like the second time I read as well. I'll keep the original and just add bytes to the list.